supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -101,7 +101,12 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
         log.error(s"The JES API worker actor $terminee unexpectedly terminated while conducting ${work.workBatch.tail.size + 1} polls. Making a new one...")
         workInProgress -= terminee
         work.workBatch.toList.foreach { toResubmit =>
-          self ! JesApiQueryFailed(toResubmit, new JesApiException(cause))
+           toResubmit match {
```

Comment:
I think you can omit `toResubmit => toResubmit match` and it will still work.  I.e. just submit the partial function